src/is-of-type.ts

```diff
@@ -0,0 +1,68 @@
+import { Action, AnyAction } from './create-action'
+import { ActionCreator } from './create-action-creator'
+import { getType } from './get-type'
+
+export function isOfType<
```

Comment:
It would be so helpful to have a comment for each case of `isOfType` function with at least a description about it and an example of using it.

Reply:
I'm not good at English, but can I still comment?

Reply:
Me too, let's try, I can help with that.

Reply:
completed!